code/products/mysql/connect.py

```diff
@@ -0,0 +1,25 @@
+import pymysql
+
+timeout = 10
+connection = pymysql.connect(
+
```

Comment:
does it make sense to remove this extra line?

Reply:
Thanks, I did that